.editorconfig

```diff
@@ -4,11 +4,22 @@ root = true
 
 [*]
 charset = utf-8
-indent_style = tab
 trim_trailing_whitespace = true
 end_of_line = lf
+indent_style = space
 insert_final_newline = true
 
 [*.py]
-indent_style = space
 indent_size = 4
+
+[*{json,yml}]
+indent_style = space
```

Comment:
I'm not sure about this because file indented with tabs is smaller than a file indented with spaces. The size of the file is important when it comes to JSON.